SQLFrontend/CommandFunctors.h

```diff
@@ -215,7 +215,8 @@ StandardCommand(Help, {
   std::cout << "\\t [regex] List all tables, optionally matching regex.\n";
   std::cout << "\\v [regex] List all views, optionally matching regex.\n";
   std::cout << "\\d <table> List all columns of a table or a view.\n";
-  std::cout << "\\c <database> <user> <password>.\n";
+  std::cout
+      << "\\c <database> <user> <password> Connect to database as different user.\n";
```

Comment:
Maybe we can use `\connect` to make it more descriptive?

Reply:
When you explained to me that `\c` does the switch, I thought that it means `change` until I saw this pull request :)

Reply:
`\connect` and `\c` both are fine with me with the new documentation.